lib/cosmos/top_level.rb

```diff
@@ -162,11 +162,12 @@ def self.data_path(name)
       # No Gemfile - so no gem based extensions
     end
 
+    # Check CORE
     filename = File.join(::Cosmos::PATH, 'data', name)
     return filename if File.exist? filename
 
-    # Check CORE
-    filename = Cosmos.path('data', name)
+    # Check relative to executing file
+    filename = Cosmos.path($0, 'config/data/' + name)
```

Comment:
I assume this is to override files in the COSMOS installation's config/data/ directory. But does this only work if the the Launcher.bat file is run from the root of the installation? What happens if you're running the bat files out of the tools directory?